### PR TITLE
feat(provider/kubernetes): token auth instructions

### PR DESCRIPTION
**Pull request by lwander**

### CODE REVIEW

File: setup/install/providers/kubernetes-v2/index.md [hunk 1]
```diff
@@ -42,6 +42,75 @@ The Kubernetes provider has two requirements:
...
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+- kind: ServiceAccount
+  name: spinnaker-service-account
+  namespace: spinnaker
+```
+
+> Note, this grants Spinnaker full access to the Kubernetes cluster using the
+> `cluster-admin` role binding.
```
Comment (dorbin) [inline setup/install/providers/kubernetes-v2/index.md]: Can we create a k8s service account with a role other than `cluster-admin`? 
IOW, is this sentence just for this *example*, or do we always give Spinnaker full access when creating a K8s serivce account?

File: setup/install/providers/kubernetes-v2/index.md [hunk 2]
```diff
@@ -42,6 +42,75 @@ The Kubernetes provider has two requirements:
...
+TOKEN=$(kubectl get secret --context $CONTEXT \
+   $(kubectl get serviceaccount spinnaker-service-account \
+       --context $CONTEXT \
+       -n spinnaker \
+       -o jsonpath='{.secrets[0].name}') \
+   -n spinnaker \
+   -o jsonpath='{.data.token}')
+```
+
+Place this token into your kubeconfig under a new user called
```
Comment (dorbin) [inline setup/install/providers/kubernetes-v2/index.md]: Backticks for `kubeconfig`?

**Review by dorbin — APPROVED** [review]
Again, just a couple of questions but lgtm.

File: setup/install/providers/kubernetes-v2/index.md (hunk 1 above)
Comment (lwander) [reply setup/install/providers/kubernetes-v2/index.md]: Good point, this is just an example. I will clarify.

**Pull request merged**